package.json

```diff
@@ -133,8 +133,8 @@
     "is-valid-domain": "0.1.2",
     "lint-staged": "10.4.0",
     "lodash": "4.17.21",
-    "lottie-ios": "3.2.3",
-    "lottie-react-native": "4.0.2",
+    "lottie-ios": "3.1.8",
```

Comment:
does this need to be downgraded ?

Reply:
Probably not, but like I mentioned I couldn't get cocoa to work without downgrading. Will remove from the PR since I didn't end up getting it to work anyways 😅